### PR TITLE
Add close

### PR DESCRIPTION
**Pull request by rokburgar**
- Fix scrollTo offset bug
- Add reform.close event so you can close every opened reform placement

### CODE REVIEW

File: src/selectbox_abstract.coffee [hunk 1]
```diff
@@ -108,6 +108,8 @@ class SelectBoxAbstract
         # Replicate changes from the original select box to the fake one
         @orig.on 'reform.sync change DOMSubtreeModified', @refresh
         
+        @body.on 'reform.close', => @close()
```
Comment (gregorpraprotnik) [inline src/selectbox_abstract.coffee]: Why do you want to close all reforms at once? I think you should have this event on `@orig` element. A reform instance should not care about other instances.

File: src/selectbox_abstract.coffee [hunk 2]
```diff
@@ -116,7 +118,7 @@ class SelectBoxAbstract
         $item.addClass 'hover'
     
     scrollTo: ($item) ->
-        $container   = @list
+        $container   = @$list
```
Comment (gregorpraprotnik) [inline src/selectbox_abstract.coffee]: :+1:

File: src/selectbox_abstract.coffee (hunk 1 above)
Comment (rokburgar) [inline src/selectbox_abstract.coffee]: I have a situation on the preview. If you drag a panel every open div on the panel should be closed else it just stays on the screen... We kinda do the same with `reform.open`.

Comment (rokburgar) [inline src/selectbox_abstract.coffee]: I I do it on orig I would have to iterate trough every reformed and trigger it..

**Pull request merged**